quill-core/src/main/scala/io/getquill/ast/Ast.scala

```diff
@@ -17,7 +17,7 @@ sealed trait Ast {
 
 sealed trait Query extends Ast
 
-case class Entity(name: String, alias: Option[String] = None, properties: List[PropertyAlias] = List()) extends Query
+case class Entity(name: String, alias: Option[String] = None, properties: List[PropertyAlias] = List(), generated: Option[String] = None) extends Query
```

Comment:
I don't know, isn't it just adding another layer inside Entity with the same properties?